sdk/keyvault/keyvault-keys/package.json

```diff
@@ -41,13 +41,13 @@
     "check-format": "prettier --list-different --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "clean": "rimraf dist dist-esm dist-test types *.tgz *.log dist-browser statistics.html coverage && rimraf src/**/*.js && rimraf test/**/*.js",
     "execute:js-samples": "node ../../../common/scripts/run-samples.js samples/javascript/",
-    "execute:ts-samples": "node ../../../common/scripts/run-samples.js samples/typescript/dist/samples/typescript/src/",
+    "execute:ts-samples": "node ../../../common/scripts/run-samples.js samples/typescript/dist/keyvault-keys/samples/typescript/src/",
```

Comment:
this too

Reply:
same https://github.com/Azure/azure-sdk-for-js/pull/9584/#discussion_r442410349